swagger.yaml

```diff
@@ -5428,6 +5430,34 @@ definitions:
     items:
       type: string
       pattern: '^([0-9]{1,3}\.){3}[0-9]{1,3}\/[0-9]|[1-2][0-9]|3[0-2]?$'
+  
+  download_boot_artifacts_request:
+    type: object
+    description: Information sent to the agent for downloading artifacts to boot a host into discovery.
+    required:
+      - kernel_url
+      - rootfs_url
+      - initrd_url
+    properties:
+      kernel_url:
+        type: string
+        description: URL address to download the kernel.
+      rootfs_url:
+        type: string
+        description: URL address to download the rootfs.
+      initrd_url:
+        type: string
+        description: URL address to download the initrd.
+
+  download_boot_artifacts_response:
```

Comment:
Do we want/need a similar request or response for the reboot step?

@avishayt ? Is that a convention we want to continue even if the step is incredibly simple?

Reply:
step-reply already defines exit_code, output, and error.  Not all steps replies define their own response and it's fine not to.  In the worst case, we now have the ability to upgrade agents and can always expand it later if necessary.